util/ubuntu-packages.txt

```diff
@@ -52,8 +52,8 @@ libprotobuf-dev
 graphviz
 
 # AVR
-avr-libc
-avrdude
+#avr-libc
+#avrdude
```

Comment:
Can we just delete these outright? There's no point in them existing in this repo.

Reply:
This is part of the 19.04 things. But yeah